### PR TITLE
Sailthru exclusion

### PR DESCRIPTION
**Pull request by alloy**
Closes https://github.com/artsy/collector-experience/issues/240

From Sailthru support:

> There is a potential to do some white/blacklisting in the App Site Association File based on our link rewrites. The hash after `/click/` is a base64 encode, and you could encode the paths accordingly before adding them to the file

I have verified that this indeed correctly excludes editorial content.

### CODE REVIEW

File: add-sailthru-paths.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+ *
+ * http://link.artsy.net/click/9466122.343103/aHR0cHM6Ly93d3cuYXJ0c3kubmV0L2FydGljbGUvYXJ0c3ktZWRpdG9yaWFsLW1ldHMtcm9vZi1ob3N0cy1kaW5uZXItcGFydHktNS0wMDAteWVhcnMtYXJ0LWludml0ZWQ_dXRtX3NvdXJjZT1zYWlsdGhydSZ1dG1fbWVkaXVtPWVtYWlsJnV0bV9jYW1wYWlnbj05NDY2MTIyLUVkaXRvcmlhbC0wNC0yNS0xNyZ1dG1fdGVybT1BcnRzeVRvcFN0b3JpZXNXZWVrbHk/56a63f2b17893fbc408b97e9B85ab6cf0
+ *
+ * (This specific link redirects to https://www.artsy.net/article/artsy-editorial-mets-roof-hosts-dinner-party-5-000-years-art-invited?utm_source=sailthru&utm_medium=email&utm_campaign=9466122-Editorial-04-25-17&utm_term=ArtsyTopStoriesWeekly)
+ *
+ * The problem with that is that iOS won’t be able to exclude paths in the normal way. However, it turns out that the
+ * encoded URL contains a base64 version of the actual Artsy URL, so we can exlcude based on that. This script takes the
+ * patterns specified in the apple-app-site-association.json file and generates the encoded versions of them. Or, if a
+ * pattern was removed, it removes the corresponding encoded pattern.
+ */
```
Comment (kanaabe) [inline add-sailthru-paths.js:12]: Interesting!

File: apple-app-site-association.json [hunk 2]
```diff
@@ -19,25 +15,11 @@
...
-      {
-        "appID": "ZH8TGFYKDK.net.artsy.artsy.beta",
-        "paths": [
-                   "NOT /article/*",
-                   "NOT /2016-year-in-art",
-                   "*"
+          "NOT /article/*",
+          "NOT /2016-year-in-art",
+          "NOT *L2FydGljbGUv*",
+          "NOT *LzIwMTYteWVhci1pbi1hcnQ=*",
```
Comment (kanaabe) [inline apple-app-site-association.json:21]: What is this one?

Comment (alloy) [reply apple-app-site-association.json:21]: It’s the base64 encoded version of `/2016-year-in-art`.

**Comment by kanaabe** [comment]
👍  Cool, thanks for catching this!

**Comment by artsyit** [comment]
:rocket: PR was released in [`v1.1.0`](https://github.com/artsy/artsy-eigen-web-association/releases/tag/v1.1.0) :rocket:

**Pull request merged**